Bejelentkezve MongooseBOT#3650 néven!
Hi Mr. Török!
|---------------------------|

Csatlakozva a(z) Developer-ra
Ignoring exception in command muzsikat:
Traceback (most recent call last):
  File "C:\Users\Simon\AppData\Local\Programs\Python\Python38\lib\site-packages\discord\ext\commands\bot.py", line 892, in invoke
    await ctx.command.invoke(ctx)
  File "C:\Users\Simon\AppData\Local\Programs\Python\Python38\lib\site-packages\discord\ext\commands\core.py", line 790, in invoke
    await self.prepare(ctx)
  File "C:\Users\Simon\AppData\Local\Programs\Python\Python38\lib\site-packages\discord\ext\commands\core.py", line 751, in prepare
    await self._parse_arguments(ctx)
  File "C:\Users\Simon\AppData\Local\Programs\Python\Python38\lib\site-packages\discord\ext\commands\core.py", line 679, in _parse_arguments
    kwargs[name] = await self.transform(ctx, param)
  File "C:\Users\Simon\AppData\Local\Programs\Python\Python38\lib\site-packages\discord\ext\commands\core.py", line 516, in transform
    raise MissingRequiredArgument(param)
discord.ext.commands.errors.MissingRequiredArgument: song is a required argument that is missing.
